From cf77a512da1ea5c2cfc36e2ba16f05765d1199aa Mon Sep 17 00:00:00 2001
From: Rob Johnston <rob.johnston@joyent.com>
Date: Thu, 25 Apr 2019 21:06:14 +0000
Subject: [PATCH] OS-7026 Add DDI support for Upgradable Firmware Modules
 Reviewed by: Robert Mustacchi <robert.mustacchi@joyent.com> Approved by:
 Jerry Jelinek <jerry.jelinek@joyent.com>

---
 overlay/generic/etc/name_to_major | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index f63a4cea..557ae7de 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -305,3 +305,5 @@ ppt 307
 coretemp 308
 amdf17nbdf 309
 pchtemp 310
+ufm 311
+ufmtest 312
-- 
2.21.0

